src/book/02-system/02-subsystems/06-behaviour.mdx

```diff
@@ -8,7 +8,58 @@ slug: /system/subsystems/behaviour
+| <kbd>d</kbd>       | Adds -0.1 to the walk command y-value. This value is in meters/second.           |
+| <kbd>z</kbd>       | Adds 0.1 to the walk command rotational value. This value is in radians/second.  |
+| <kbd>x</kbd>       | Adds -0.1 to the walk command rotational value. This value is in radians/second. |
+| <kbd>,</kbd>       | Runs the kick with the left foot.                                                |
+| <kbd>.</kbd>       | Runs the kick with the right foot.                                               |
+| <kbd>g</kbd>       | Runs the get up.                                                                 |
+| <kbd>&#8592;</kbd> | Head turns to the left.                                                          |
+| <kbd>&#8594;</kbd> | Head turns to the right.                                                         |
+| <kbd>&#8593;</kbd> | Head turns upwards.                                                              |
+| <kbd>&#8595;</kbd> | Head turns downwards.                                                            |
+| <kbd>r</kbd>       | Resets keyboardwalk. Head rotation is set to 0. Walk command is set to 0.        |
+| <kbd>q</kbd>       | Quits keyboardwalk.                                                              |
+
+To use KeyboardWalk, build the [keyboardwalk role](https://github.com/NUbots/NUbots/blob/master/roles/keyboardwalk.role) and run the binary.
+
+The [**keyboardwalkfake** role](https://github.com/NUbots/NUbots/blob/master/roles/keyboardwalkfake.role) uses HardwareSimulator, for when you don't want to run keyboard walk on the robot.
+
+### PS3 Walk
+
+The [PS3 walk module](https://github.com/NUbots/NUbots/tree/master/module/behaviour/strategy/PS3Walk) is an interface for **testing** and **demonstrating** the walk and other functionalities using a **PS3 controller**.
+
+The module connects to the PS3 controller using bluetooth (or USB) and reacts based on the joystick or button input.
+
+| Command                   | Description                                |
+| ------------------------- | ------------------------------------------ |
+| Left Joystick Horizontal  | Adds rotational speed to the walk command. |
+| Left Joystick Vertical    | Adds x speed to the walk command.          |
+| Right Joystick Horizontal | Changes the head yaw.                      |
+| Right Joystick Vertical   | Changes the head pitch.                    |
+| TRIANGLE                  | Toggles walking on/off.                    |
+| SQUARE                    | Toggles locking of the head.               |
+| R2                        | Right kick.                                |
+
+To use this module, build the [PS3Walk role](https://github.com/NUbots/NUbots/blob/master/roles/ps3walk.role) and run the binary. Plug the PS3 controller into the robot using a USB cable. When the controller is connected to the robot, you can remove the cable.
```

Comment:
```suggestion
To use this module, build the [PS3Walk role](https://github.com/NUbots/NUbots/blob/master/roles/ps3walk.role) and run the binary. Plug the PS3 controller into the robot using a USB cable. When the controller is connected to the robot, it will be paired, and you can remove the cable.
```